scripts/mail-alarm

```diff
@@ -34,66 +34,43 @@ FromLineOverride=YES
 ma_username="__dom0__mail_alarm"
 
 mail_language_pack_path = '/etc/xapi.d/mail-languages/'
```

Comment:
Please include the ticket number in the commit title, it makes finding it later much easier.